File: front-end/src/app/service-models/app/v1/dummy-subject-s-m.ts
import { CoinManagementServiceModelBase } from '../base/coin-management-service-model-base';

export class DummySubjectSM extends CoinManagementServiceModelBase<number> {
    subjectName!: string;
    subjectCode!: string;
    dummyTeacherID?: number;
}

File: backend/payroll-backend/Components/Siffrum.Web.Payroll.ServiceModels/v1/DummySubjectSM.cs
using Siffrum.Web.Payroll.ServiceModels.Base;

namespace Siffrum.Web.Payroll.ServiceModels.v1
{
    //[IgnoreClassAutoMap]
    public class DummySubjectSM : SiffrumPayrollServiceModelBase<int>
    {
        public string SubjectName { get; set; }
        public string? SubjectCode { get; set; }
        public int? DummyTeacherID { get; set; }
    }
}
